notebooks/data-sources/TestGrid/testgrid_EDA.ipynb

```diff
@@ -13,14 +13,28 @@
     "\n",
```

Comment:
<p>I think the typo fix here should be "I believe this outputs states..." </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/91/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>